kong.yml

```diff
@@ -90,7 +91,7 @@ nginx: |
     real_ip_recursive on;
 
     # Other Settings
-    client_max_body_size 128m;
+    client_max_body_size 0m;
```

Comment:
NGINX doesn't allow overriding when a value is set, thus any plugin being enabled to secure an API will not be applied.

Removing the value, and adding a security suggestion within the documentation alerting the user to this will be sufficient enough and allow us to deploy it. From my understanding the principals surrounding KONG is a way for the user to avoid any complex or having to modify the NGINX configuration in any fashion and this is a way towards that principal.

Something else I think that would be beneficial in the future is some form of pre-defined configurations based on #295 that would alleviate any question of security missteps in the community.